subsys/bluetooth/host/l2cap.c

```diff
@@ -3068,6 +3070,20 @@ int bt_l2cap_chan_disconnect(struct bt_l2cap_chan *chan)
 	return 0;
 }
 
+__maybe_unused static bool user_data_not_empty(const struct net_buf *buf)
+{
+	size_t ud_len = sizeof(struct closure);
+	uint8_t *ud = net_buf_user_data(buf);
```

Comment:
Ew, `net_buf_user_data` casts away the `const` from `buf`. 

Nevertheless, this could be
```suggestion
	const uint8_t *ud = net_buf_user_data(buf);
```

but is OK to keep